presale/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+{
+    'name': "Presale Orders",
+    'version': '1.0.0',
```

Comment:
_NOTE_
Not mandatory but we usually use the full version number `16.0.1.0.0` where `16.0` is the version of Odoo the task was made for and `1.0.0` is the version of the current module